### ISSUE TITLE
Escaping of violation.message

### ISSUE THREAD

**Issue by whitty** [tomasbjerre/violation-comments-to-bitbucket-server-lib#10]
I'm not sure that this is logged in the correct place: but

According to my reading of https://github.com/tomasbjerre/violation-comments-lib/blob/master/src/main/java/se/bjurr/violations/comments/lib/model/ViolationData.java

via the README at https://github.com/tomasbjerre/violation-comments-lib the violation message is always escaped as HTML.  Is it possible to override part of this to support bitbucket-server contents which write *markdown* in their comments:

## Details

I have `commentTemplate` defined as follows (the default, but made explicit)

```
 commentTemplate: '''**Reporter**: {{violation.reporter}}{{#violation.rule}}

**Rule**: {{violation.rule}}{{/violation.rule}}
**Severity**: {{violation.severity}}
**File**: {{violation.file}} L{{violation.startLine}}{{#violation.source}}

**Source**: {{violation.source}}{{/violation.source}}

{{violation.message}}''',
```

However the HTML escaping shows poorly in bitbucket comments - in this case `'` single quotes are being escaped into numeric escapes which are (a) ugly and (b) linked to unrelated pull-request of the same id as the escape :-/

see below

![violation-escape](https://user-images.githubusercontent.com/194537/57013846-c3393300-6c50-11e9-98c6-00d26660bdbd.png)

Have I misunderstood? can the unescaped code be inserted somehow?

If you need to "escape" for markdown you could consider simply wrapping it in \`\`\` \`\`\` and leaving the escaping to the markdown processor.

**Comment by whitty** [comment]
Sorry - I think this is closed by tomasbjerre/violation-comments-lib#10
 
The confusion is from the README in violation-comments-lib, which directs to ViolationData.java for reference.  I think that has been refactored into almost irrelevance (I think it should point to violation-lib/.../Violation.java directly).  But reading through naively without understanding it all I could see was the escape handling in that file.

Now that you've identified mustache I can see that the escaping is coming from there: `{{violation.message}}` is an escaped string.

I re-wrote the template:

```
    commentTemplate: '''**Reporter**: {{violation.reporter}}{{#violation.rule}}

**Rule**: {{violation.rule}}{{/violation.rule}}
**Severity**: {{violation.severity}}
**File**: {{violation.file}}:{{violation.startLine}}{{#violation.source}}

**Source**: {{violation.source}}{{/violation.source}}

```
{{{violation.message}}}
```''',
```

Which seems to work as required, although I guess it could have potential xss issues if the tool output message were not nicely constrained.

**Comment by tomasbjerre** [comment]
Ok!
